test/__mocks__/pack/mockPack.json

```diff
@@ -0,0 +1 @@
+{"url":"https://freesound.org/people/reinsamba/packs/9678/","description":"","created":"2012-05-23T22:25:41","name":"Zoo","username":"reinsamba","num_sounds":2,"num_downloads":37}
```

Comment:
We should format all these json files with prettier so they're human readable:

```
npx prettier --write test/__mocks__/*
```

Reply:
Thank you for the command! I believe 1256708 addresses this.

Reply:
🥳 remember to resolve corresponding comments

Reply:
Yup! I think I've been doing that. It might be a glitch on my end.

![Screen Shot 2021-04-27 at 7 47 08 PM](https://user-images.githubusercontent.com/43425812/116338908-5b5f4980-a791-11eb-9cfd-82d1b6b2ed93.png)

![Screen Shot 2021-04-27 at 7 49 57 PM](https://user-images.githubusercontent.com/43425812/116339129-c01aa400-a791-11eb-86c2-ec769ce2cba8.png)

![Screen Shot 2021-04-27 at 7 50 29 PM](https://user-images.githubusercontent.com/43425812/116339208-e0e2f980-a791-11eb-91b4-6a8caed48249.png)